frontend/src/utils/random_data.ts

```diff
@@ -0,0 +1,69 @@
+export const NICKNAMES = [
+  "Person_A",
+  "Person_B",
+  "Person_C",
+  "Person_D",
+  "Person_E",
+  "Person_F",
+  "Person_G",
+  "Person_H",
+  "Person_I",
+  "Person_J",
+  "Person_K",
+  "Person_L",
+  "Person_M",
+  "Person_N",
+  "Person_O",
+  "Person_P",
+  "Person_Q",
+  "Person_R",
+  "Person_S",
+  "Person_T",
+  "Person_U",
+  "Person_V",
+  "Person_W",
+  "Person_X",
+  "Person_Y",
+  "Person_Z",
+]
+
+export const PRONOUNS = [
+  "He/Him",
+  "She/Her",
+  "They/Them",
+  "Xe/Xem",
+  "Ze/Zem",
+]
+
+export const EMAILS = [
+  "Email_A",
+  "Email_B",
+  "Email_C",
+  "Email_D",
+  "Email_E",
+  "Email_F",
+  "Email_G",
+  "Email_H",
+  "Email_I",
+  "Email_J",
+  "Email_K",
+  "Email_L",
+  "Email_M",
+  "Email_N",
+  "Email_O",
+  "Email_P",
+  "Email_Q",
+  "Email_R",
+  "Email_S",
+  "Email_T",
+  "Email_U",
+  "Email_V",
+  "Email_W",
+  "Email_X",
+  "Email_Y",
+  "Email_Z",
+]
+
+export function pickRandom<T>(options: T[]): T {
+  return options[Math.floor(Math.random() * options.length)];
+}
```

Comment:
```suggestion
}

```
Newline at end of file.